WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/Categories/ProductCategoryListViewModelTests.swift

```diff
@@ -121,10 +77,10 @@ final class ProductCategoryListViewModelTests: XCTestCase {
                 exp.fulfill()
             }
         }
-        wait(for: [exp], timeout: Constants.expectationTimeout)
+        waitForExpectations(timeout: Constants.expectationTimeout, handler: nil)
```

Comment:
My bad. I actually meant the one in [this file](https://github.com/woocommerce/woocommerce-ios/blob/develop/Networking/NetworkingTests/Testing/XCTestCase+Wait.swift). But now I realize that it's not `public`. 🤦 I think we should move that to a Core/Utilities framework that shares all our useful utils. Someday.